quantities/src/main/java/com/decathlon/vitamin/compose/quantities/VitaminQuantities.kt

```diff
@@ -0,0 +1,238 @@
+package com.decathlon.vitamin.compose.quantities
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminQuantities {
+
+    @Suppress("LongMethod")
+
+    @Composable
+    fun Default(
```

Comment:
According to the Figma specification, you should add an error mode for your component.

![Capture d’écran 2022-11-16 à 15 56 14](https://user-images.githubusercontent.com/1913901/202214757-9779b3b9-6521-4a07-b456-6d5f99956a3d.png)

Reply:
j'ai ajouté un borderColorError et un boolean hasQuantityError pour afficher le textField en erreur. Est ce qu'il manque quelque chose ? je vois pas c'est quoi comme état le bleu